pkg/apis/starrocks/v1/load_type.go

```diff
@@ -84,6 +85,12 @@ type StarRocksLoadSpec struct {
 	// the reference for configMap which store the config info to start starrocks. e.g. be.conf, fe.conf, cn.conf.
 	// +optional
 	ConfigMapInfo ConfigMapInfo `json:"configMapInfo,omitempty"`
+
+	// StartupProbeFailureSeconds defines the total failure seconds of startupProbe.
+	// By default, the startupProbe use HTTP /api/health Probe, the failureThreshold is 60, the periodSeconds is 5.
```

Comment:
default failureThreshold is 60 and the periodSeconds is 5, this means the startup will fail if the pod can't start in 300 seconds. Your StartupProbeFailureSeconds is the total time of seconds before startupProbe give up and fail the container start.